cloudify_types/cloudify_types/password/operations.py

```diff
@@ -0,0 +1,107 @@
+from random import choice, shuffle
```

Comment:
for security purposes, `random.choice` and `random.shuffle` aren't enough, because their output is predictable. `random.SystemRandom` would be better.

Python3 also exposes `choice` in the `secrets` module, usefully - but alas, not `shuffle`.